README.md

```diff
@@ -54,6 +54,7 @@ Parameter             | Description
 space                 | Hash with an user choosen name for the space as key and the space id as value
 access_token          | Contentful Delivery API access token
 cda_query             | Hash describing query configuration. See [contentful.rb](https://github.com/contentful/contentful.rb) for more info (look for filter options there). Note that by default only 100 entries will be fetched, this can be configured to up to 1000 entries using the `limit` option. Example: `f.cda_query = { limit: 1000 }`
+cda_query             | Hash describing client configuration. See [contentful.rb](https://github.com/contentful/contentful.rb) for more info (look for client configuration there). This option should commonly be used to change Rate Limit Management, Include Resolution, Logging and Proxies.
```

Comment:
Maybe using [https://github.com/contentful/contentful.rb#client-configuration-options](https://github.com/contentful/contentful.rb#client-configuration-options) instead of suggesting to search?